.github/workflows/branch-protection.yml

```
@@ -0,0 +1,99 @@
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  check-branch-naming:
+    name: Check Branch Naming Convention
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Check branch name
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)